server/pbench/bin/pbench-base.sh

```diff
@@ -77,14 +77,14 @@ function mk_dirs {
     hostname=$1
 
     for d in $LINKDIRS ;do
-        thedir=$ARCHIVE/$hostname/$d
+        thedir=$ARCHIVE/hostnames/$hostname/$d
```

Comment:
Wait - this will apply to the new structure only, which means we have to have a complete setup of the old structure *left intact* so that it can deal with old agents. We already made a distinction between new and old scripts (e.g. pbench-dispatch-inotify and pbench-dispatch). If you make a change like this, you are breaking the old scripts.